Payments/UserInterface/01a_ListAuthorizations.md

```diff
@@ -0,0 +1,142 @@
+[!!Monitor authorizations](../Operation/01_ManageAuthorizations.md)
+
+# LIST (Authorizations)
+
+![LIST (Authorizations)](../../Assets/Screenshots/Payments/Authorizations/LISTAuthorizations.png "[LIST (Authorizations)]")
+
+**Authorizations**
+
+- *VIEW*  
+    Click the drop-down list to select the view. All created views are displayed in the drop-down list. Click the ![Points](../../Assets/Icons/Points01.png "[Points]") (Points) button to the right of the *VIEW* drop-down list to display the context menu and create a view.   
+
+    - ![Points](../../Assets/Icons/Points01.png "[Points]") (Points)      
+        Click this button to the right of the *View* drop-down list to display the context menu. The following menu entries are available:
+
+        - ![Create](../../Assets/Icons/Plus06.png "[Create]") create  
+            Click this entry to create a view. The *Create view* window is displayed, see [Create view](#create-view).
+
+        - ![Rename](../../Assets/Icons/Edit02.png "[Rename]") rename  
+            Click this entry to rename the selected view. The *Rename view* window is displayed, see [Rename view](#rename-view). This menu entry is only displayed if you have selected a view.
+
+        <!---- Eintrag gibt es nicht mehr [Reset](../../Assets/Icons/Reset.png "[Reset]") reset  
+            Click this entry to reset all unsaved changes to the settings of the selected view. This menu entry is only displayed if a view has been selected and any changes have been made to the view settings.--->
+
+        - ![Publish](../../Assets/Icons/Publish.png "[Publish]") publish  
+            Click this entry to publish the view. This menu entry is only displayed if you have selected an unpublished view. As long as you have not published a view, your views are visible for you only.
+
+        - ![Unpublish](../../Assets/Icons/Unpublish.png "[Unpublish]") unpublish  
+            Click this entry to unpublish the view so that it is visible only to you again. This menu entry is only displayed if you have selected and published a view.
```

Comment:
```suggestion
            **Click this entry to unpublish the view so that it is visible only to you again**. This menu entry is only displayed if you have selected and published a view.
```
Ist es so? Visible for you only vs. visible für andere Mitarbeiter? Hast du das mit Stefan geklärt? Wenn ja, sollten wir das mal aufschreiben und die Änderungen durchgängig machen (VIEW Menü kommt in anderen Modulen vor).